client/landing/jetpack-cloud/sections/settings/main.jsx

```diff
@@ -69,11 +67,10 @@ const ConnectionStatus = ( { isConnected } ) => {
 	);
 };
 
-const SettingsPage = () => {
+export const SettingsPage = () => {
 	const translate = useTranslate();
-	const siteId = useSelector( getSelectedSiteId );
-
-	const rewind = useSelector( ( state ) => getRewindState( state, siteId ) );
+	const siteId = useSelectedSiteId();
+	const rewind = useRewindState( siteId );
```

Comment:
One approach I see a lot is that we export the component and then as the default export the component wrapped in `connect` from the Redux package. Would that approach work here instead? I'm worried about the complexity of maintaining a separate file for tests.

Reply:
I share your concern, it doesn't seem viable. Using `connect` instead of hooks would definitely work, but testing considerations should probably not drive this kind of decision. Unfortunately, exporting these selectors from the same file as the component won't work with Jest, because selectors need to be mocked while the component needn't.

After doing some more reading, trying to do shallow testing with components that use selectors seems a bit like trying to fit a square into a circle. I'd like to read about other approaches that exist out there. I believe we can close this PR.